tdvt/tdvt/exprtests/standard/expected.setup.cast.str.14.txt

```diff
@@ -0,0 +1,163 @@
+<results>
+  <test name='STR([int1])'>
+    <sql>
+      SELECT CAST(`Calcs`.`int1` AS STRING) AS `TEMP_Test__2617331766__0_`&#10;FROM `tactile-pulsar-824`.`TestV1`.`Calcs` `Calcs`&#10;GROUP BY 1&#10;    </sql>
+    <query-time>
+      527.000&#10;    </query-time>
+    <table>
+      <schema>
+        <column>[cast_calcs.googlebigquery].[TEMP(Test)(2617331766)(0)]</column>
+      </schema>
+      <tuple>
+        <value>%null%</value>
+      </tuple>
+      <tuple>
+        <value>&quot;-3&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;-4&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;-6&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;-8&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;-9&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;2&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;3&quot;</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='STR([bool0])'>
+    <sql>
+      SELECT CAST(`Calcs`.`bool0` AS STRING) AS `TEMP_Test__3200082645__0_`&#10;FROM `tactile-pulsar-824`.`TestV1`.`Calcs` `Calcs`&#10;GROUP BY 1&#10;    </sql>
+    <query-time>
+      570.000&#10;    </query-time>
+    <table>
+      <schema>
+        <column>[cast_calcs.googlebigquery].[TEMP(Test)(3200082645)(0)]</column>
+      </schema>
+      <tuple>
+        <value>%null%</value>
+      </tuple>
+      <tuple>
+        <value>&quot;false&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;true&quot;</value>
+      </tuple>
```

Comment:
I believe this is a bug. I looked at all the other expected files and its by design that casting `STR([bool0])` should be null/0/1

Reply:
I added `CAP_QUERY_BOOLEXPR_TO_INTEXPR=true`, but it seems like still getting true/false. Is there any places that i should take a look?

Reply:
I made a note in the other review about where you could change the dialect if you wanted to verify the cause of this. Let's handle it as follow-up since we'll want to fork the dialect.